watertap/unit_models/zero_order/sedimentation_zo.py

```diff
@@ -61,3 +62,53 @@ def rule_basin_surface_area(b, t):
                         to_units=pyunits.ft**2))
         self.basin_surface_area_constraint = Constraint(
             self.flowsheet().time, rule=rule_basin_surface_area)
+
+        if self.config.process_subtype == "phosphorus_capture":
+            self.phosphorus_solids_ratio = Var(self.flowsheet().config.time,
+                                               units=pyunits.dimensionless,
+                                               doc="Mass fraction of phosphorus in settleable solids")
+
+            self._fixed_perf_vars.append(self.phosphorus_solids_ratio)
+            self._perf_var_dict["Phosphorus-Solids Ratio (kg/kg)"] = self.phosphorus_solids_ratio
+
+            if ("phosphates" in self.config.property_package.solute_set
+                or "phosphate_as_phosphorus" in self.config.property_package.solute_set) \
+                    and "tss" in self.config.property_package.solute_set:
+                raise KeyError("tss and phosphates cannot both be defined in the solute_list. "
```

Comment:
I was aiming to be more restrictive on the use of this subtype since it should really just be specified for phosphorous capture. Moreover, if the user provides TSS, the amount of settled phosphate would be determined based on an assumed fraction of phosphate in TSS. Alternatively, the user could provide phosphates as the species, and the amount of solids + phosphate settled would be reported. However, the user cannot provide both.  I can add a comment describing such.